#!/bin/sh /usr/share/dpatch/dpatch-run
## 40-kolabd-pid-file.dpatch by Noèl Köthe <noel@debian.org>
##
## DP: This is the patch to correct the pid file path

@DPATCH@
--- kolabd-1.9.4/kolabd.orig	2006-01-25 16:32:11.000000000 +0100
+++ kolabd-1.9.4/kolabd	2006-01-25 16:32:46.000000000 +0100
@@ -52,8 +52,8 @@
 openlog('kolabd', 'cons, pid', 'user');
 
 my $prefix = $Kolab::config{'prefix'};
-my $pidfile = IO::File->new("$prefix/var/kolab/kolab.pid", 'w+')
-    || die "Unable to open PID file `$prefix/var/kolab/kolab.pid'";
+my $pidfile = IO::File->new("/var/run/kolab.pid", 'w+')
+    || die "Unable to open PID file `/var/run/kolab.pid'";
 print $pidfile $$;
 undef $pidfile;
 
